Sources/MapboxNavigation/OrnamentsController.swift

```diff
@@ -110,7 +110,7 @@ extension NavigationMapView {
             guard navigationView.resumeButton.isHidden else { return }
             
             if let roadName = roadName {
-                navigationView.wayNameView.text = roadName
+                navigationView.wayNameView.text = roadName.isEmpty ? nil : roadName
```

Comment:
```suggestion
                navigationView.wayNameView.text = roadName.nonEmptyString
```

Reply:
Thanks for the suggestion, I'll improve this one.